examples/lit-ts/stories/SimpleGreeting.ts

```diff
@@ -0,0 +1,15 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import style from './simplegreeting.scss';
```

Comment:
I'm not sure why, but I'm getting a typescript error in my editor on this line.  Are you seeing the same?